### PR TITLE
[SPARK-28652][TESTS][K8S] Add python version check for executor

### PR DESCRIPTION
**Pull request by viirya**
## What changes were proposed in this pull request?

Current two PySpark version tests in PythonTestsSuite, just test against Python version at driver side. Because the test script doesn't run any spark job requiring python worker, it doesn't actually do version check at worker side. This patch adds pieces of code to the test script, to run a simple job to verify Python version.

## How was this patch tested?

Unit test. Locally manual test.

### CODE REVIEW

**Comment by viirya** [comment]
This patch was coming from my test on this code, in order to look into the original ticket SPARK-28652. I think it is just for test coverage.

**Comment by viirya** [comment]
cc @HyukjinKwon @skonto

File: resource-managers/kubernetes/integration-tests/tests/pyfiles.py [hunk 1]
```diff
@@ -35,4 +36,11 @@
     # Begin of Python container checks
     version_check(sys.argv[1], 2 if sys.argv[1] == "python" else 3)
 
+    # Check python executable at executors
+    spark.catalog.registerFunction("getSysVer",
+                                   lambda: "%d.%d" % sys.version_info[:2], StringType())
+    [row] = spark.sql("SELECT getSysVer()").collect()
+    driverVersion = "%d.%d" % sys.version_info[:2]
```
Comment (HyukjinKwon) [inline resource-managers/kubernetes/integration-tests/tests/pyfiles.py]: shall we stick to `driver_version` name? `getSysVer` -> `get_sys_ver` too

File: resource-managers/kubernetes/integration-tests/tests/pyfiles.py [hunk 2]
```diff
@@ -35,4 +36,11 @@
     # Begin of Python container checks
     version_check(sys.argv[1], 2 if sys.argv[1] == "python" else 3)
 
+    # Check python executable at executors
+    spark.catalog.registerFunction("getSysVer",
```
Comment (HyukjinKwon) [inline resource-managers/kubernetes/integration-tests/tests/pyfiles.py]: Let's use `spark.udf.register`. `spark.catalog.registerFunction` is deprecated as of SPARK-23122

**Review by HyukjinKwon — APPROVED** [review]

File: resource-managers/kubernetes/integration-tests/tests/pyfiles.py (hunk 1 above)
Comment (viirya) [reply resource-managers/kubernetes/integration-tests/tests/pyfiles.py]: Yes.

**Comment by HyukjinKwon** [comment]
Merged to master.

**Pull request closed without merging**